manifests/t/Tencent/QQBrowser/10.8.4455.400/Tencent.QQBrowser.installer.yaml

```
@@ -0,0 +1,46 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Tencent.QQBrowser
+PackageVersion: 10.8.4455.400
+MinimumOSVersion: 10.0.0.0
+FileExtensions:
+- doc
+- docx
+- htm
+- html
+- mht
+- mhtml
+- pdf
+- ppt
+- pptx
+- shtml
+- xht
+- xhtml
+- xls
+- xlsx
+Protocols:
+- ftp
+- http
+- https
+InstallerSuccessCodes:
+- 4294967295
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: zh-CN
+  Architecture: x86
```

```suggestion
-  Architecture: x86
```
It looks like we need to start to move away from using `InstallerLocale` here because this prevents machines outside of the `zh-CN` locale from upgrading the application using the `winget upgrade` command.

See https://github.com/microsoft/winget-pkgs/issues/25347#issuecomment-903033816.